docs/first_pr.md

```diff
@@ -0,0 +1,61 @@
+# Getting started with EvalAI Issues:
```

Comment:
Also instead of making this document as first pull request, lets make it `pull-request` and then create a section called `Making your First Pull request`. This way we can add more sections in the future.